addons/mail/static/src/new/messaging_menu/messaging_menu.xml

```diff
@@ -41,7 +41,7 @@
                 </div>
             </t>
         </div>
-        <div t-if="state.tab !== 'mailbox'" class="d-flex flex-column overflow-auto flex-grow-1">
+        <div t-if="(props.inDiscuss and ['chat', 'channel'].includes(store.discuss.activeTab)) or !props.inDiscuss" class="d-flex flex-column overflow-auto flex-grow-1">
```

Comment:
Can be simplified to `!props.inDiscuss or ['chat', 'channel'].includes(store.discuss.activeTab)`

Reply:
I actually removed `props.inDiscuss` and instead use `env.inDiscussApp` (it's former `env.inDiscuss`, with suffix `App` for clarity)